lib/validation.ex

```diff
@@ -44,12 +44,11 @@ defmodule TdDfLib.Validation do
          changeset,
          %{"depends" => %{"on" => depend_on, "to_be" => depend_to_be}} = content_item
        ) do
-    case Map.get(changeset.data, depend_on) do
-      ^depend_to_be ->
-        add_content_validation(changeset, Map.drop(content_item, ["depends"]))
-
+    dependent_value = Map.get(changeset.data, depend_on)
```

Comment:
should use https://hexdocs.pm/ecto/Ecto.Changeset.html#get_change/3 or https://hexdocs.pm/ecto/Ecto.Changeset.html#get_field/3?

Reply:
indeed.